src/ProgramSequence.js

```diff
@@ -156,7 +156,25 @@ export default class ProgramSequence {
 
     overwriteStep(index: number, command: string): ProgramSequence {
```

Comment:
Is there any reason not to type this as `CommandName` now that we have it?

Reply:
We discussed in chats, we don't currently have a set of values for what's used here (and in the program block value).